content/4_Gold/Tree_Euler.problems.json

```
@@ -58,6 +58,18 @@
     }
   ],
   "problems": [
+    {
+      "uniqueId": "cses-1139",
+      "name": "Distinct Colors",
+      "url": "https://cses.fi/problemset/task/1139",
+      "source": "CSES",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Euler Tour", "BIT/Merge_Sort_Tree"],
```

```suggestion
      "tags": ["Euler Tour"],
```
removing this bc theres many ways to get distinct #s in a range